### DISCUSSION TITLE
Basic spatial unit for LCZ results

### DISCUSSION THREAD

**Discussion by atoulou** [orbisgis/geoclimate#959] [Q&A] [answered]
Hi there,
I was wondering if it would be possible to set a minimum size for the generation of the RSUs, which are then the spatial units used for LCZ classification? 
If it is possible, how can we do it (what do we have to change/add in the code?)
Thanks in advance!

**Comment by j3r3m1** [comment, marked as answer]
Hi @atoulou 
It is not yet possible to set a minimum size in the RSU generation. However, it is possible to use the grid indicators using a specific grid resolution to obtain cells at the size you like. Just note that using this technic, the LCZ type attributed to your cell will be the mode (the most represented LCZ type from your RSU) within your cell. So it is a kind of aggregation of the information.
